core/order.go

```diff
@@ -211,7 +261,16 @@ func prepareModeratedOrderContract(data *PurchaseData, n *OpenBazaarNode, contra
 	payment.Address = addr.EncodeAddress()
 	payment.RedeemScript = hex.EncodeToString(redeemScript)
 	payment.Chaincode = hex.EncodeToString(chaincode)
-	contract.BuyerOrder.RefundFee = wal.GetFeePerByte(wallet.NORMAL)
+	fee := wal.GetFeePerByte(wallet.NORMAL)
+	contract.BuyerOrder.RefundFeeValue = &pb.CurrencyValue{
+		Currency: &pb.CurrencyDefinition{
+			Code:         data.PaymentCoin,
```

Comment:
Use `defn` for value in case we decide to normalize through this abstraction later.